connectors/db/vector.py

```diff
@@ -143,6 +143,10 @@ def search(self, query, search_filter, query_embedding) -> list[SearchResult]:
         * https://github.com/pgvector/pgvector-python/tree/master/examples/hybrid_search
         * https://python.langchain.com/docs/how_to/hybrid/
         """
+        if not cfg.ENABLE_HYBRID_SEARCH:
+            log.debug("Hybrid search is disabled")
```

Comment:
Can we move this log message into `__init__` so it is only logged once at startup?